hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/clientrm/TestFederationClientInterceptor.java

```diff
@@ -1404,10 +1390,12 @@ public void testSubmitReservationMultipleSubmission() throws Exception {
     Assert.assertNotNull(subClusterId1);
     Assert.assertTrue(subClusters.contains(subClusterId1));
 
-    // First Retry
+    // First Retry, Repeat the submission
```

Comment:
lower case for repeat

Reply:
Thank you very much for your help reviewing the code, I will modify the code.